.coverage.sh

```diff
@@ -10,25 +10,18 @@ if [ -z "$KEEP" ]; then trap "popd; rm -rf $COVERAGE_DIR" EXIT; fi
 
 $(which cp) -r ../* .
 
-eval `opam config env`
-opam install -y bisect_ppx oasis ocveralls
+export COVERAGE=1
+jbuilder runtest
```

Comment:
We still need to install the packages: `opam install -y bisect_ppx oasis ocveralls`